src/Switch.tsx

```diff
@@ -23,7 +23,7 @@ export const fieldToSwitch = ({
     ...props,
     ...field,
     // TODO: is this the correct way?
-    value: field.value ? 'checked' : '',
+    checked: field.value,
```

Comment:
I did some research on this and I now think it should be:
```
value: field.name,
checked: field.value
```

Reply:
Thanks, I just made that correction.
In fact, I only changed and verified that the switch started working correctly.